api/src/cli/utils/create-env/env-stub.liquid

```diff
@@ -1,60 +1,331 @@
+# KEY="xxxxxxx-xxxxxx-xxxxxxxx-xxxxxxxxxx"
+
+# Secret string for the project 
+# SECRET="abcdef"
+
+# The duration that the access token is valid ["15m"]
 ACCESS_TOKEN_TTL="15m"
+
+# The duration that the refresh token is valid, and also how long users stay logged-in to the App ["7d"]
 REFRESH_TOKEN_TTL="7d"
+
+# Whether or not to use a secure cookie for the refresh token in cookie mode ["false"]
 REFRESH_TOKEN_COOKIE_SECURE=false
+
+# Value for sameSite in the refresh token cookie when in cookie mode ["lax"]
 REFRESH_TOKEN_COOKIE_SAME_SITE="lax"
+
+# Name of refresh token cookie ["directus_refresh_token"]
 REFRESH_TOKEN_COOKIE_NAME="directus_refresh_token"
 
+# Which domain to use for the refresh cookie. Useful for development mode.
+# REFRESH_TOKEN_COOKIE_DOMAIN
+
+# Whether or not to enable the CORS headers [false]
+CORS_ENABLED="true"
+
+# Value for the Access-Control-Allow-Origin header. Use true to match the Origin header, or provide a domain or a CSV of domains for specific access [false]
+CORS_ORIGIN="true"
+
+# Value for the Access-Control-Allow-Methods header [GET,POST,PATCH,DELETE]
+CORS_METHODS=GET,POST,PATCH,DELETE
+
+# Value for the Access-Control-Allow-Headers header [Content-Type,Authorization]
+CORS_ALLOWED_HEADERS=Content-Type,Authorization
+
+# Value for the Access-Control-Expose-Headers header [Content-R
+CORS_EXPOSED_HEADERS=Content-Range
+
+# Whether or not to send the Access-Control-Allow-Credentials header [true]
+CORS_CREDENTIALS="true"
+
+# Value for the Access-Control-Max-Age header [18000]
+CORS_MAX_AGE=18000
+
+####################################################################################################
+### Argon2
+
+# How much memory to use when generating hashes, in KiB [4096]
+# HASH_MEMORY_COST=81920
+
+# The length of the hash function output in bytes [32]
+# HASH_HASH_LENGTH=32
+
+# The amount of passes (iterations) used by the hash function. It increases hash strength at the cost of time required to compute [3]
+# HASH_TIME_COST=10
+
+# The amount of threads to compute the hash on. Each thread has a memory pool with HASH_MEMORY_COST size [1]
+# HASH_PARALLELISM=2
+
+# The variant of the hash function (0: argon2d, 1: argon2i, or 2: argon2id) [1]
+# HASH_TYPE=2
+
+An extra and optional non-secret value. The value will be included B64 encoded in the parameters portion of the digest []
+# HASH_ASSOCIATED_DATA=foo
+
 ####################################################################################################
-## Auth Providers
+### Auth Providers
 
+# A comma-separated list of auth providers []
 AUTH_PROVIDERS=""
+# AUTH_PROVIDERS="github"
+
+# AUTH_GITHUB_DRIVER="oauth2"
+# AUTH_GITHUB_CLIENT_ID="73e...4b"
+# AUTH_GITHUB_CLIENT_SECRET="b9...98"
+# AUTH_GITHUB_AUTHORIZE_URL="https://github.com/login/oauth/authorize"
+# AUTH_GITHUB_ACCESS_URL="https://github.com/login/oauth/access_token"
+# AUTH_GITHUB_PROFILE_URL="https://api.github.com/user"
+# AUTH_GITHUB_ALLOW_PUBLIC_REGISTRATION="false"
```

Comment:
Think it makes more sense to have this option "true" by default?

Reply:
> # This allows users to be automatically created on logins. Use "false" if you want to create users manually

I copied the example.env file but I agree! Thanks